Bottom: 1ced3a9f220fdf407b23aec49ccc32df78590dd3
Top:    eb6746be1d3ee5ea492f24a09e25d5ab0601a3d5
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-30 19:19:37 +0800

Refresh of autofs-5.0.7-teach-dumpmaps-to-output-maps-to-files.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index faa2e69..6d08749 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -59,7 +59,7 @@
 - dont start readmap unless ready.
 - fix dumpmaps multi output.
 - try and cleanup after dumpmaps.
-- teach dumpmaps to output map files.
+- teach dumpmaps to output maps to files.
 
 25/07/2012 autofs-5.0.7
 =======================
